modules/auxiliary/scanner/mssql/mssql_version.rb

```diff
@@ -0,0 +1,63 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Auxiliary
+  include Msf::Exploit::Remote::MSSQL
+  include Msf::Auxiliary::Scanner
+  include Msf::OptionalSession::MSSQL
+
+  def initialize
+    super(
+      'Name' => 'MSSQL Version Utility',
+      'Description' => 'Executes a TDS7 pre-login request against the MSSQL instance to query for version information.',
+      'Author' => 'Zach Goldman',
+      'License' => MSF_LICENSE
+    )
+
+    register_options([
+      Opt::RPORT(1433)
+    ])
+  end
+
+  def run
+    if session
+      set_session(session.client)
```

Comment:
Not a blocker; we'll still want to rename this on our side - i.e. `set_mssql_session` or such